<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DanmuWindow</class>
 <widget class="QWidget" name="DanmuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DanmuDisplay</string>
  </property>
  <property name="windowOpacity">
   <double>0.700000000000000</double>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>331</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_content">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QListWidget" name="list_danmu">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_liveRoomInfo">
      <item>
       <widget class="QLabel" name="prompt_followersCount">
        <property name="text">
         <string>关注人数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_followersCount">
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="prompt_viewersCount">
        <property name="text">
         <string>直播人气</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_viewersCount">
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
